code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/ldap/OpenLDAP/OpenLDAPIdentityProvider.java

```diff
@@ -123,7 +123,7 @@
                             '=' + memberOf.get(i).toString() + ")(" + getConstantsConfig().objectClass() + '=' +
                             getConstantsConfig().getGroupObjectClass() + "))";
                     logger.trace("getIdentities: memberOf attribute query: "+query);
-                    identities.addAll(resultsToIdentities(searchLdap(query)));
+                    identities.addAll(resultsToIdentities(searchLdap(query, "user")));
                 }
```

Comment:
Lets also support this feature for OpenLDAP, since we are introducing the code to search groups under a groupSearchDomain in a common place (LDAPIdentityProvider), we can as well expose this in API and support it for both AD and OpenLDAP